src/instrument/utils/dm_metadata_file.py

```diff
@@ -0,0 +1,140 @@
+"""
+Write the metadata file for processing a bluesky run with a DM workflow.
+
+All metadata content comes from the bluesky run's metadata dictionary. The
+bluesky session is responsible to provide all expected metadata keys.
+"""
+
+import logging
+import pathlib
+
+import databroker
+import h5py
+import yaml
+
+from .conversions import ts2iso
+from .iconfig_loader import iconfig  # noqa
+
+CATALOG = iconfig.get("DATABROKER_CATALOG", "8idi_xpcs")
+XPCS_LAYOUT_VERSION = "APS-8IDI-202310"
```

Comment:
What does this showcase?

Reply:
`XPCS_LAYOUT_VERSION` is for the consumer of this file, to expect that certain items are present.

Reply:
Nice, is there anywhere for users to see what layout versions are expected and the options one could use?

Reply:
No, this is a conversation with @AZjk .